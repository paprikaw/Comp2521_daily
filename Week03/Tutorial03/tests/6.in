4 6 5 7 8 10 55 67 68 50 11