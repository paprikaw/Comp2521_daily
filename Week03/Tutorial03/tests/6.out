Tree:
4
 \
  6
 / \
5   7
     \
      8
       \
       10
         \
         55
         / \
        /   \
       50   67
      /       \
     11       68

#nodes = 11,  #leaves = 3
Infix  : 4 5 6 7 8 10 11 50 55 67 68 
Prefix : 4 6 5 7 8 10 55 50 11 67 68 
Postfix: 5 11 50 68 67 55 10 8 7 6 4 
ByLevel: 4 6 5 7 8 10 55 50 67 11 68 
